feature_engine/selection/__init__.py

```diff
@@ -1,7 +1,7 @@
 """
 The module selection includes classes to select features or remove unwanted features.
 """
-
+from .eliminate_feature_by_feature_importance import RecursiveFeatureElimination
```

Comment:
Could we change the name of the .py to recursive_feature_elimination.py ?

The reason is that we (will) have several transformers that select features by importance, and it might be confusing to understand which one we will find in this script if we keep the generic name.

Thank you!